websites/S/Sofi/presence.ts

```diff
@@ -0,0 +1,164 @@
+const presence = new Presence({
+		clientId: "1028679580027977839",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	content = {
+		ViewingImage: "https://i.imgur.com/9WPWqvT.png",
+		BuyingImage: "https://i.imgur.com/JKLNlvT.png",
+		ViewingImageText: "Viewing...",
+		BuyingImageText: "Buying...",
+	};
```

Comment:
I think a buy icon and viewing icon are quite self explanatory so the text is not needed. Also, the a `enum` for the `Assets `

Reply:
What do you mean by "a `enum` for the `Assets`. Also, I would prefer removing the images rather than the text.

Reply:
This
https://github.com/PreMiD/Presences/blob/01054ec857bc5052c45a0ee39a044ce21dd0bf85/websites/M/MangaFreak/presence.ts#L4-L8